src/lib/handlers/TrustPingHandler.ts

```diff
@@ -0,0 +1,31 @@
+import { Handler } from './Handler';
+import { InboundMessage } from '../types';
+import { TrustPingService } from '../protocols/trustping/TrustPingService';
+import { ConnectionService } from '../protocols/connections/ConnectionService';
+import { MessageType } from '../protocols/trustping/messages';
+
+export class TrustPingHandler implements Handler {
+  trustPingService: TrustPingService;
+  connectionService: ConnectionService;
+
+  constructor(trustPingService: TrustPingService, connectionService: ConnectionService) {
+    this.trustPingService = trustPingService;
+    this.connectionService = connectionService;
+  }
+
+  async handle(inboundMessage: InboundMessage) {
+    switch (inboundMessage.message['@type']) {
+      case MessageType.TrustPingMessage:
+        const { recipient_verkey } = inboundMessage;
+        const connection = this.connectionService.findByVerkey(recipient_verkey);
+        if (!connection) {
+          throw new Error(`Connection for receipient_verkey ${recipient_verkey} not found`);
+        }
+        return this.trustPingService.process_ping(inboundMessage, connection);
+      case MessageType.TrustPingResponseMessage:
+        return this.trustPingService.process_ping_response(inboundMessage);
+      default:
+        throw new Error('Invalid message type for handler');
+    }
+  }
```

Comment:
uhm no problem as such but I figured it's easy to see all handlers for a particular protocol at one place rather than switching b/w files.

You're right that this breaks consistency with what we have at the moment. I'll split this into two different handlers. Maybe we can have another issue that refactors the handlers and groups them by protocol type? If so, could you create one please?